backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Api.Database.Models;
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
```

Comment:
```suggestion
using Api.Database.Models;
using System.Timers;
```

Reply:
Add namespace for timers

Reply:
'Timer' is an ambiguous reference between 'System.Timers.Timer' and 'System.Threading.Timer' [api]csharp(CS0104)

For this reason I have to specfically import in this manner and thus the import statement on the top is not required.

Reply:
Ah ok, makes sense!